application/language/spanish/options_lang.php

```diff
@@ -2,125 +2,125 @@
+$lang['options_this_text_is_an_optional_text_that_can_be_displayed_on_top_of_the_oqrs_page'] = 'Este texto es un texto opcional que se mostrará en la parte superior de la página de OQRS.';
+$lang['options_grouped_search'] = 'Búsqueda agrupada';
+$lang['options_when_this_is_on_all_station_locations_with_oqrs_active_will_be_searched_at_once'] = 'Cuando está activo, se buscarán en simultáneo todas las localizaciones de estación con OQRS activo';
+$lang['options_grouped_search_show_station_name'] = "Mostrar localización de la estación en los resultados de búsqueda agrupados";
+$lang['options_grouped_search_show_station_name_hint'] = "Si la búsqueda agrupada esta ACTIVA, puede decidir si el nombre de la localización de la estación se omsytrará en la tabla de resultados.";
+$lang['options_oqrs_options_have_been_saved'] = 'Las opciones de OQRS se han guardado.';
 
 $lang['options_dxcluster'] = 'DXCluster';
-$lang['options_dxcluster_provider'] = 'Provider of DXClusterCache';
-$lang['options_dxcluster_longtext'] = 'The Provider of the DXCluster-Cache. You can set up your own Cache with <a href="https://github.com/int2001/DXClusterAPI">DXClusterAPI</a> or use a public one';
-$lang['options_dxcluster_hint'] = 'URL of the DXCluster-Cache. e.g. https://dxc.jo30.de/dxcache';
+$lang['options_dxcluster_provider'] = 'Proveedor de DXClusterCache';
+$lang['options_dxcluster_longtext'] = 'El proveedor de DXCluster-Cache. Puede configurar su propio caché con <a href="https://github.com/int2001/DXClusterAPI">DXClusterAPI</a> o usar un caché público.';
+$lang['options_dxcluster_hint'] = 'URL del DXCluster-Cache. ej. https://dxc.jo30.de/dxcache';
 $lang['options_dxcluster_settings'] = 'DXCluster';
-$lang['options_dxcache_url_changed_to'] = 'DXCluster Cache URL changed to ';
-$lang['options_dxcluster_maxage'] = 'Maximum Age of spots taken care of';
-$lang['options_dxcluster_maxage_hint'] = 'The Age in Minutes of spots, that will be taken care at bandplan/lookup';
-$lang['options_dxcluster_decont'] = 'Show spots which are spotted from following continent';
-$lang['options_dxcluster_maxage_changed_to']='Maximum age of spots changed to ';
-$lang['options_dxcluster_decont_changed_to']='de continent changed to ';
-$lang['options_dxcluster_decont_hint']='Only spots by spotters from this continent are shown';
+$lang['options_dxcache_url_changed_to'] = 'URL del DXCluster-Cache cambiada a ';
+$lang['options_dxcluster_maxage'] = 'Máxima edad de puntos a tomarse en cuenta';
+$lang['options_dxcluster_maxage_hint'] = 'La edad en minutos de los puntos que se tomarán en cuenta en el plan de bandas/búsqueda';
+$lang['options_dxcluster_decont'] = 'Mostrar puntos que se observen en el siguiente continente';
+$lang['options_dxcluster_maxage_changed_to']='Máxima edad de puntos cambiada a ';
+$lang['options_dxcluster_decont_changed_to']='de continente cambiado a ';
+$lang['options_dxcluster_decont_hint']='Solo se muestran puntos por observadores de este continente';
```

Comment:
I would leave "spots" in English. That's how people usually call them, even when speaking Spanish. If translating it into Spanish, the translation it's certainly not "puntos".